EmuLibrary/Settings/SettingsView.xaml

```diff
@@ -9,166 +9,164 @@
-            </TextBlock>
-        </Grid>
-        <DataGrid Grid.Row="1" Padding="0,0,0,4" ItemsSource="{Binding Mappings}" AutoGenerateColumns="False" CellEditEnding="DataGrid_CellEditEnding" CurrentCellChanged="DataGrid_CurrentCellChanged" SelectionUnit="Cell"
-                  CanUserAddRows="True">
-            <DataGrid.Resources>
-                <ObjectDataProvider x:Key="RomTypeValues"
-                                    ObjectType="{x:Type sys:Enum}"
-                                    MethodName="GetValues">
-                    <ObjectDataProvider.MethodParameters>
-                        <x:Type TypeName="elrt:RomType" />
-                    </ObjectDataProvider.MethodParameters>
-                </ObjectDataProvider>
-            </DataGrid.Resources>
-            <DataGrid.Columns>
-                <DataGridTemplateColumn Width="50">
-                    <DataGridTemplateColumn.CellTemplate>
-                        <DataTemplate>
-                            <Button Content="❌" Tag="{Binding}" Click="Click_Delete" Foreground="Red" Width="26" Margin="0" Padding="0" />
-                        </DataTemplate>
-                    </DataGridTemplateColumn.CellTemplate>
-                    <DataGridTemplateColumn.CellEditingTemplate>
-                        <DataTemplate><TextBlock Text=""></TextBlock></DataTemplate>
-                    </DataGridTemplateColumn.CellEditingTemplate>
-                </DataGridTemplateColumn>
-                <DataGridTemplateColumn Header="Emulator" Width="180">
-                    <DataGridTemplateColumn.CellTemplate>
-                        <DataTemplate>
-                            <TextBlock x:Name="Emulator" Text="{Binding Path=Emulator.Name, UpdateSourceTrigger=PropertyChanged}" />
-                        </DataTemplate>
-                    </DataGridTemplateColumn.CellTemplate>
-                    <DataGridTemplateColumn.CellEditingTemplate>
-                        <DataTemplate>
-                            <ComboBox Name="ComboEmulators" DockPanel.Dock="Left"
-                                      ItemsSource="{Binding AvailableEmulators}"
-                                      DisplayMemberPath="Name"
-                                      SelectedItem="{Binding Emulator, Mode=TwoWay}"
-                                      >
-                            </ComboBox>
-                        </DataTemplate>
-                    </DataGridTemplateColumn.CellEditingTemplate>
-                </DataGridTemplateColumn>
+    <DockPanel Margin="20">
+        <ScrollViewer DockPanel.Dock="Top">
+            <StackPanel x:Name="EmulatorSettingsPanel">
+                <Label Content="Emulator path mappings" FontWeight="Bold" />
+                <TextBlock HorizontalAlignment="Right" Margin="0,-16,0,0">
+                    <Hyperlink NavigateUri="https://github.com/psychonic/Playnite-EmuLibrary/blob/master/README.md#setup" RequestNavigate="Hyperlink_RequestNavigate">

```

Comment:
I also noticed that this link didn't work, so I've pointed it at the correct README.md URL.

Reply:
Good catch